docs/howto/smart_contract_upload.md

```diff
@@ -0,0 +1,156 @@
+# Smart Contract Upload
+
+This tutorial covers how to create a circuit on a splinter network between 
+two nodes and upload a smart contract via the circuit.
+
+## Prerequisites
+
+* A splinter network with at least two nodes running. The tutorial will refer
+  to the two nodes as "alpha" and "beta" respectively
+* The scabbard command line tool installed
+* The splinter command line tool installed
+* A compiled smart contract. Check [here](https://sawtooth.hyperledger.org/docs/sabre/nightly/master/application_developer_guide.html#) for details on how to create and compile a smart contract
+
+## Procedure
+
+1. Create a new circuit
```

Comment:
This is going to change as soon as https://github.com/Cargill/splinter/pull/492 goes in. Should this PR wait until that is merged and the information added can be more up to date?

Reply:
I wouldn't have a problem waiting until 492 goes in to merge this.

Reply:
The PR was merged, now we just need the splintercommunit/splinterd:experimental docker image to be published during the nightly build, and the new way to create circuits will work.

Reply:
Updated with cli command